contrib/build-linux/appimage/apprun.sh

```diff
@@ -3,9 +3,14 @@
 set -e
 
 APPDIR="$(dirname "$(readlink -e "$0")")"
+PYTHON="${APPDIR}/usr/bin/python3.6"
 
 export LD_LIBRARY_PATH="${APPDIR}/usr/lib/:${APPDIR}/usr/lib/x86_64-linux-gnu${LD_LIBRARY_PATH+:$LD_LIBRARY_PATH}"
 export PATH="${APPDIR}/usr/bin:${PATH}"
 export LDFLAGS="-L${APPDIR}/usr/lib/x86_64-linux-gnu -L${APPDIR}/usr/lib"
 
-exec "${APPDIR}/usr/bin/python3.6" -s "${APPDIR}/usr/bin/electron-cash" "$@"
+if ! "$PYTHON" -s "${APPDIR}/test-freetype.py" ; then
+    export LD_LIBRARY_PATH="${APPDIR}/usr/lib/fonts${LD_LIBRARY_PATH+:$LD_LIBRARY_PATH}"
```

Comment:
Works for me! 👍

Reply:
I needed this because if the system has a too new fontconfig, the included fontconfig can have problems parsing the configuration files.

Reply:
Ah makes sense. Nice that you caught that.